adaptors/backbone/backbone_view_widget.js

```diff
@@ -140,20 +140,20 @@ BackboneViewWidget.prototype.update = function update(prev, elem) {
   }
 };
 
-/* develblock:start */
+if (typeof TUNGSTENJS_DEBUG_MODE !== 'undefined') {
```

Comment:
Pedantic, but is it feasible to always define this either as true or false so we can just do `if (TUNGSTEN_JS_DEBUG_MODE)` rather than the undefined check?